colossalai/inference/config.py

```diff
@@ -85,3 +94,27 @@ def _verify_config(self) -> None:
         assert (
             self.tp_size * self.pp_size == dist.get_world_size()
         ), f"TP size({self.tp_size}) * PP size({self.pp_size}) should be equal to the global world size ({dist.get_world_size()})"
+
+    def _get_tokenizer(self) -> None:
+        if self.tokenizer is not None:
+            return
+        model_name = self.model.lower()
+        if "llama" in model_name:
+            self.tokenizer = "hf-internal-testing/llama-tokenizer"
+
+    def _to_generation_config(self, model_config) -> GenerationConfig:
```

Comment:
If this method is supposed to be exposed for other classes to call, it should just not start with `_`, it will be just good to use `to_generation_config`.